.github/workflows/cd.yml

```diff
@@ -98,30 +105,25 @@ jobs:
           git config --global user.email 'yiz@microsoft.com'
 
       - name: release alpha npm packages to npmjs.org
-        if: ${{ github.event_name == 'schedule' }}
+        if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/dev' && github.event.inputs.preid == 'alpha')}}
         run: |
           npx lerna version --no-private prerelease --preid=alpha.$(git rev-parse --short HEAD) --no-push --allow-branch dev --yes
           npx lerna publish from-package --dist-tag=alpha --yes --allow-branch dev
 
-      - name: release rc npm packages to npmjs.org
-        if: ${{ github.event_name == 'workflow_dispatch' && startsWith(github.event.inputs.version, 'pre') && github.event.inputs.conventional-commits == 'no'}}
+      - name: version rc npm packages to npmjs.org
+        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'rc' && github.event.inputs.skip-rc-version == 'no'}}
         run: |
-          npx lerna publish --no-private --preid=${{ github.event.inputs.preid }} --dist-tag=${{ github.event.inputs.preid }} ${{ github.event.inputs.version }} --no-changelog --yes ${{ github.event.inputs.force }}
+          npx lerna version --no-private --conventional-commits --conventional-prerelease --preid=rc --no-changelog --yes
 
-      - name: release stable npm packages to npmjs.org
-        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && !startsWith(github.event.inputs.version, 'pre') && github.event.inputs.conventional-commits == 'no'}}
-        run: |
-          npx lerna publish --no-private --dist-tag=latest ${{ github.event.inputs.version }} --no-changelog --yes ${{ github.event.inputs.force }}
-
-      - name: release rc npm packages to npmjs.org by conventional-commits
-        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.conventional-commits == 'yes' && github.event.inputs.preid == 'rc'}}
+      - name: publish rc npm packages to npmjs.org
+        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'rc' }}
```

Comment:
do we need to check `skip-rc-version` here?

Reply:
skip-rc-verison is optional, not required, so there is no need to check